app/main/containers/Search/index.js

```diff
@@ -369,7 +369,7 @@ Search.propTypes = {
 function mapStateToProps(state) {
   return {
     selected: state.search.selected,
-    results: state.search.resultIds.map(id => state.search.resultsById[id]),
+    results: state.search.resultIds.map(id => state.search.resultsById[id]).sort((a, b) => b.selectCount - a.selectCount),
```

Comment:
This sort will be applied for all results, not only from `basic-apps`, so would be better to move it to plugin